app/about/PhilosophySection.tsx

```
@@ -0,0 +1,30 @@
+        {Philosophy.map((item, index) => (
+          <div
+            key={index}
+            className={`p-8 rounded-lg shadow-lg flex flex-col gap-4 ${
+              index === Philosophy.length - 1 &&
+              index % 2 === 0 &&
+              "md:col-span-2"
+            }`}
+          >
+            <h3 className="text-3xl font-bold text-brand">{item.title}</h3>
+            <p className="text-gray-300 lg:text-lg">
+              <span dangerouslySetInnerHTML={{ __html: item.description }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)